src/components/Hoverable/index.js

```diff
@@ -22,11 +22,9 @@ class Hoverable extends Component {
      */
     toggleHoverState() {
         if (this.state.isHovered) {
-            this.props.onHoverOut();
-            this.setState({isHovered: false});
+            this.setState({isHovered: false}, this.props.onHoverOut());
         } else {
-            this.props.onHoverIn();
-            this.setState({isHovered: true});
+            this.setState({isHovered: true}, this.props.onHoverIn());
```

Comment:
We need to do one of these instead as passing a called function will execute it immediately and not in the callback 

```js
this.setState({isHovered: true}, this.props.onHoverIn);
```

or

```js
this.setState({isHovered: true}, () => this.props.onHoverIn());
```

Reply:
Good call